bokeh/embed/server.py

```diff
@@ -170,6 +170,9 @@ def server_session(model, session_id, url="default", relative_urls=False, resour
             probably not desired.
 
     '''
+    if session_id is None:
+        raise ValueError("Must supply a session_id")
+
```

Comment:
This would be great with py3 keyword only args, some day...

Also I'd like to make `server_session(session=mysession)` possible, it will complicate arg handling but I think it is the natural API we missed along the way. Ideally it would have been `server_session(mysession)` but that ship has sailed